changelogs/fragments/9537-gitlab-fix-api-compatibility.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - gitlab_* modules - improve compatibility for different API URLs when using gitlab-python 3.8.0 or newer (https://github.com/python-gitlab/python-gitlab/pull/2149).
```

The URL should be the URL of this very PR. See references below.
```suggestion
  - gitlab_* modules - improve compatibility for different API URLs when using gitlab-python 3.8.0 or newer (https://github.com/python-gitlab/python-gitlab/pull/2149).
```

Please refer to the standard formatting for the changelog fragments' entries:

https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to-format

The outcome of that can be seen in the collection changelog:

https://docs.ansible.com/ansible/latest/collections/community/general/changelog.html

---

I think the suggestion should be:
```suggestion
  - gitlab_* modules - improve compatibility for different API URLs when using gitlab-python 3.8.0 or newer (https://github.com/ansible-collections/community.general/pull/9537).
```
:)